CHANGELOG.md

```diff
@@ -9,6 +18,13 @@
 * Enhancement - [Knob to enable bandwidth plugin](https://github.com/aws/amazon-vpc-cni-k8s/pull/1580) (#1580, [@jayanthvn](https://github.com/jayanthvn))
 * Testing - [IPv6 Integration test suite](https://github.com/aws/amazon-vpc-cni-k8s/pull/1658) (#1658, [@achevuru](https://github.com/achevuru))
 
+## v1.9.3
```

Comment:
Are we adding this now?

Reply:
I noticed that this had got out of sync with master. So added it